join.html

```diff
@@ -407,6 +407,11 @@ <h2 class="progress">Here is your personalized link</h2>
               joining link within 7 working days.
             </li>
           </ol>
+          <div class="goto-home-button-container">
+            <a class="button-filled goto-home-btn" href="./goto.html"
```

Comment:
Just a minor suggestion ,we have a class button-filled but it is not a filled button. It should be button-outline

Reply:
yes @khushi818 thanks